src/OrchardCore/OrchardCore/Shell/Configuration/ShellConfigurationSources.cs

```diff
@@ -70,5 +73,27 @@ public async Task SaveAsync(string tenant, IDictionary<string, string> data)
                 }
             }
         }
+
+        public Task RemoveAsync(string tenant)
+        {
+            var tenantFolder = Path.Combine(_container, tenant);
+            var appsettings = Path.Combine(tenantFolder, "appsettings.json");
```

Comment:
Isn't this logic defined elsewhere in the code? maybe we should have a helper to create the path to the appsettings?